schemas/bindings.gschema.xml

```diff
@@ -79,6 +79,83 @@
         Toggle the material-shell UI to simulate fullscreen
       </description>
     </key>
+    <key name="use-maximize-layout" type="as">
+      <default><![CDATA[['<Super>g']]]></default>
+      <summary>Use maximize layout</summary>
+      <description>
+        Use maximize layout
+      </description>
+    </key>
+    <key name="use-split-layout" type="as">
+      <default><![CDATA[['<Super>z']]]></default>
```

Comment:
Isn't Super+Z already used for moving to the last workspace? I think maybe something like Ctrl + Super + <Layout Name Initial> would be easier to remember, at least for the main layouts: Maximize, Split, Half, Ratio, Float and Simple.

Reply:
Yeah you are right, I am changing that, thank you :)